daemon/pool.go

```diff
@@ -402,26 +402,26 @@ func (p *movingAverage) Scale(total, load int) int {
 	}
 
 	avg := sum / float64(len(p.loads))
-	return p.ScalingPolicy.Scale(total, int(avg))
+	return p.sub.Scale(total, int(avg))
 }
 
 type minMax struct {
-	ScalingPolicy
+	Sub      ScalingPolicy
```

Comment:
I'm not entirely clear on why the name of this field is `sub`. When it was unexported it didn't really matter, but now that it's exported it probably deserves either a comment or maybe a more descriptive name. :)

Reply:
Unexported and commented on the field.